src/descriptions.js

```diff
@@ -75,5 +75,13 @@ module.exports = {
   jack: "During the day, you are able to select two people whether they would trick or treat and which option would kill them by using `+trickorteat <player1> <player2> <trick/treat>`. Your aura is **Unknown** and you are a **Solo** team player.",
   "pumpkin king": "Once per game, you can go trick or treating by giving your candy bucket to a dead player using `+bucket <player>`. That player will add a candy into the bucket and then either pass it on to another dead player, or give it back to you. Once the candy bucket has been returned, all players who have given candy will receive a small gift. Your aura is **Good** and you are a **Village** team player.",
   modded: "You are a modded role. When you receive this role, you will get further instructions from the narrator on what you are supposed to do.",
-  "astral wolf": "You are the Astral Wolf. During the day, you can give any non-wolf a blessing with [command], granting them the ability to see someone else's role on the next night. After you give your blessing, you may mark 3 players using [command]. If any one of those marked players die, the remaining marked players will die with them."
+  "astral wolf": "You are the Astral Wolf. During the day, you can give any non-wolf a blessing with [command], granting them the ability to see someone else's role on the next night. After you give your blessing, you may mark 3 players using [command]. If any one of those marked players die, the remaining marked players will die with them.",
```

Comment:
You are the Astral Wolf. During the day, you can give any non-wolf a blessing with [command], granting them the ability to see someone else's role the next night. After you give your blessing, you may mark three players using [command]. If any one of those marked players dies, the remaining marked players will die with them.